mteb/tasks/Audio/Clustering/eng/VehicleSoundClustering.py

```diff
@@ -17,7 +17,7 @@ class VehicleSoundClustering(AbsTaskAudioClustering):
         category="a2a",
         eval_splits=["test"],
         eval_langs=["eng-Latn"],
```

Comment:
Probably not English
```suggestion
        eval_langs=[],
```
Not sure if this is valid though

Reply:
probably worth going through all the clustering datasets just to check this

Reply:
Wait so if it's not English, what folder should we put these tasks in? Right now they're in `eng` 🤔

Reply:
I'd imagine this applies to bioacoustics and cityscapes etc as well